Modix/Utilities/LimitToChannelsAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Discord.Commands;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Modix.Utilities
+{
+    public class LimitToChannelsAttribute : PreconditionAttribute
+    {
+        public override Task<PreconditionResult> CheckPermissions(ICommandContext context, CommandInfo command, IDependencyMap map)
+        {
+            var channelsConfig = Environment.GetEnvironmentVariable($"MODIX_LIMIT_MODULE_CHANNELS_{command.Module.Name.ToUpper()}");
```

Comment:
... use the DAL maybe? If you dont want to install postgress, just change the data adapter for EF core to in-memory for your copy, it´ll be fine. Other than that, LGTM! :)